docs/detections/alerts-view-details.asciidoc

```diff
@@ -125,31 +173,81 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
+
+[discrete]
+[[threat-match-detected-section]]
+==== Threat match detected
+
+The Threat match detected section is solely reserved for threat indicator matches identified by an indicator match rule. Threat indicator matches are produced whenever event data matches a threat indicator field value in your indicator index. 
+
+[discrete]
+[[enriched-with-threat-intelligence-section]]
+==== Enriched with threat intelligence
+
+The Enriched with threat intelligence section shows indicator matches that {elastic-sec} found when querying the alert for fields with threat intelligence. You can use the date time picker to modify the query time frame, which looks at the past 30 days by default. 
+
+Click the **Inspect** button, located on the far right of the threat label, to view more information on the query.
+
+NOTE: The event enrichment query uses the indices specified in the `securitySolution:defaultThreatIndex` advanced setting. For more information, refer to <<update-threat-intel-indices, Update default Elastic Security threat intelligence indices>>.
+
+[discrete]
+[[correlations-section]]
+=== Correlations 
 
-All other available threat intelligence data appears in the **Enriched with Threat Intelligence** subsection.
+The Correlations sub-section shows how an alert is related to other alerts and offers ways to investigate related alerts. Use this information to quickly find patterns between alerts and then take action. 
+
+The Correlations sub-section provides the following information: 
+
+* **Suppressed alerts**: Indicates that the alert was created with alert suppression, and shows how many duplicate alerts were suppressed. This information only appears if alert suppression is enabled for the rule.
+* **Alerts related by source event**: Shows alerts that were created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. 
+* **Cases related to the alert**: Shows the total number and names of cases to which the alert has been added.
+* **Alerts related by session ID**: Shows alerts that generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. 
++
+NOTE: To use this feature, you must enable the **Include session data** setting in your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
+* **Alerts related by process ancestry**: Shows alerts that are related by process events on the same linear branch. When you examine this information further, note that alerts generated from processes on child or related branches are not shown. 
++
+NOTE: To access data about alerts related by process ancestry, you must have a https://www.elastic.co/pricing[Platinum or higher subscription].
+
+[discrete]
+[[prevalence-section]]
+=== Prevalence
```

Comment:
I feel like it could be valuable to have a `Prevalence overview` section here before the `Prevalence sub-tab you have`. The reason being that this section has a little bit of logic that is not intuitive to the users:
- for each highlighted field key-value pairs, we calculate the host prevalence, then only show some data if the prevalence is between 0 and 10%, in the form of a sentence that would look like this:
`field, value is uncommon`

Reply:
@PhilippeOberti does this section examine host events only? I thought both host and user events were examined in the prevalence calculations. I mainly say this because the copy we discussed in Slack with @christineweng and @paulewing indicates that both types of events are considered.

Reply:
The prevalence overview (right panel) only shows data based on host prevalence.
The prevalence details (left panel) shows alert count, document count, host prevalence and user prevalence data.